Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.yml

```diff
@@ -0,0 +1,2860 @@
+          description: user first name.
+          type: String
+        - contextPath: BmcITSM.User.LastName
+          description: user last name.
+          type: String
+      arguments:
+        - name: user_ids
+          description: Comma separated list of user ID. Filtering argument.
+          required: false
+          isArray: true
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: query
+          description: 'Search qualification to list by. For example: Status = "Draft" AND Impact = "1-Extensive/Widespread". The query will be used as an addition to the existing args. Review the BMC documentation for how to Build search qualifications: https://docs.bmc.com/docs/ars2008/building-qualifications-and-expressions-929630007.html'
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: limit
+          description: The maximum number of records to retrieve.
+          required: false
+          isArray: false
+          defaultValue: "50"
+          predefined:
+            - ""
+        - name: page_size
+          description: The maximum number of records to retrieve per page.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: page
+          description: The page number of the results to retrieve.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: first_name
+          description: The user first name. Filtering argument.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: last_name
+          description: The user first name. Filtering argument.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: company
+          description: The user company name. Filtering argument.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: department
+          description: The user department name. Filtering argument.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: organization
+          description: The user organization name. Filtering argument.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+    - name: bmc-itsm-company-list
+      description: List companies in BMC Helix ITSM. The records are retrieved according to a constraint determined either by the query argument or by the filtering arguments. When using filtering arguments, Each one defines a 'LIKE' operation and 'AND' operator is used between them.
```

Comment:
Lists companies in BMC Helix ITSM. The records are retrieved by the query argument or by the filtering arguments. When using filtering arguments, each one defines a 'LIKE' operation and the 'AND' operator is used between them.